src/content/blog/2024-10-23-rafikis-first-security-audit.mdx

```diff
@@ -0,0 +1,90 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Rafiki's First Security Audit"
+description: "Takeaways from Rafiki's 2024 security audit"
+date: 2024-10-23
+slug: rafikis-first-security-audit
+authors: [Max Kurapov]
+author_urls: [https://www.linkedin.com/in/mkurapov]
+tags:
+  - Rafiki
+  - security
+  - audit
+---
+
+At the beginning of the year, we were in contact with a security and penetration testing company to do an audit of Rafiki. Even though the software is still in its early stages, it is essential to gather feedback early to build a strong foundation for the software's security. The primary goals of the assessment were to evaluate several Rafiki components: the GraphQL Admin APIs, the frontend Admin UI component, as well as our underlying [ILPv4 protocol](https://interledger.org/developers/get-started/). The assessment was done using Rafiki’s local playground, based on the [Open Source Security Testing Methodology Manual (OSSTMM)](https://www.isecom.org/research.html) and [Open Source Web Application Security Project (OWASP)](https://owasp.org/) methodologies. 
```

Comment:
Didn't we also get our Open Payments implementation tested or did that happen afterwards?